### ISSUE TITLE
Should we generate an global enum using for all (new) enums which be use the "logic" like a numpad position? 

### ISSUE THREAD

**Issue by aismann** [adxeproject/adxe#699]
Maybe this enum should be in a global class because many parts of the engine could use this enum like anchoring and what not, something like AnchorCorner enum or EdgePosition enum.

```
/**
 * @brief NumPadPosType can be used for all situations where to set the position on object like the numbers on the numpad
 */
enum NumPadPosType
{
    BOTTOM_LEFT    = 1,
    BOTTOM_CENTER  = 2,
    BOTTOM_RIGHT   = 3, 
    CENTER_LEFT    = 4,
    CENTER         = 5,
    CENTER_RIGHT   = 6,
    TOP_LEFT       = 7,
    TOP_CENTER     = 8,
    TOP_RIGHT      = 9
};
```

Where is the right place (file) for this 'global' enum?

**Comment by DelinWorks** [comment]
i think `ccTypes.h` is a good place

**Comment by aismann** [comment]
Thanks. I will send a PR

**Comment by aismann** [comment]
> i think `ccTypes.h` is a good place

Or a new file like: 'ccEnum.h' is a better place for the future?

**Comment by DelinWorks** [comment]
that's actually a good idea, wanna do it or do i do it? you'd have to move every enum that the engine uses there

**Comment by aismann** [comment]
Maybe you are more familiar with enums on adxe.
Please add you it.
Better you are not change the existing enums at the moment because of backwards compatibility.
The only one enum should be 'FPSPosition' because it is new.

**Comment by DelinWorks** [comment]
> MaBetter you are not change the existing enums at the moment because of backwards compatibility.

This won't hurt backwards compatibility because the enums that are previously used are going to be exposed anyways but in another file.

**Comment by aismann** [comment]
I have start it (PR with no new file: ccEnum.h).
I will push it now. Ok?

**Comment by DelinWorks** [comment]
This issue can be closed now as it's dealt with in #702

**Comment by aismann** [comment]
closed
